src/styles/images.scss

```diff
@@ -26,3 +26,8 @@ $man-blue: url('../public/assets/thumbnail/man-blue.svg');
 $key-blue: url('../public/assets/thumbnail/key-blue.svg');
 $cloud-white: url('../public/assets/thumbnail/cloud-white.svg');
 $cloud-blue: url('../public/assets/thumbnail/cloud-blue.svg');
+
+/**
+ * Max icon in SendAssetAmount
+ */
+$max-blue: url('public/assets/thumbnail/max-blue.svg');
```

Comment:
would be better to use it right in its place, because we migrated to css modules, so now such images config is deprecated.